docs/subnets/create-a-simple-rust-vm.md

```diff
@@ -0,0 +1,1256 @@
+        .about("Timestamp Vm")
+        .subcommands(vec![genesis::command(), vm_id::command()])
+        .get_matches();
+
+    // ref. https://github.com/env-logger-rs/env_logger/issues/47
+    env_logger::init_from_env(
+        env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+    );
+
+    match matches.subcommand() {
+        Some((genesis::NAME, sub_matches)) => {
+            let data = sub_matches.get_one::<String>("DATA").expect("required");
+            let genesis = timestampvm::genesis::Genesis { data: data.clone() };
+            println!("{genesis}");
+
+            Ok(())
+        }
+
+        Some((vm_id::NAME, sub_matches)) => {
+            let vm_name = sub_matches.get_one::<String>("VM_NAME").expect("required");
+            let id = subnet::vm_name_to_id(vm_name)?;
+            println!("{id}");
+
+            Ok(())
+        }
+
+        _ => {
+            log::info!("starting timestampvm");
+
+            let (stop_ch_tx, stop_ch_rx): (Sender<()>, Receiver<()>) = broadcast::channel(1);
+            let vm_server = subnet::rpc::vm::server::Server::new(vm::Vm::new(), stop_ch_tx);
+            subnet::rpc::plugin::serve(vm_server, stop_ch_rx).await
+        }
+    }
+}
+```
+
+### Installing a VM
+
+AvalancheGo searches for and registers plugins under the `plugins` directory of
+the [build directory](../nodes/maintain/avalanchego-config-flags#build-directory).
+
+To install the virtual machine onto your node, you need to move the built
+virtual machine binary under this directory.  Virtual machine executable names
+must be either a full virtual machine ID (encoded in CB58), or a VM alias.
+
+Copy the binary into the plugins directory.
+
+```bash
+cp -n <path to your binary> $GOPATH/src/github.com/ava-labs/avalanchego/build/plugins/
+```
+
+#### Node Is Not Running
+
+If your node isn't running yet, you can install all virtual machines under your
+`plugin` directory by starting the node.
+
+#### Node Is Already Running
+
+Load the binary with the `loadVMs` API.
+
+```bash
+curl -sX POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"admin.loadVMs",
+    "params" :{}
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/admin
+```
+
+Confirm the response of `loadVMs` contains the newly installed virtual machine
+`tGas3T58KzdjLHhBDMnH2TvrddhqTji5iZAMZ3RXs2NLpSnhH`. You'll see this virtual
+machine as well as any others that weren't already installed previously in the
+response.
+
+```json
+{
+  "jsonrpc": "2.0",
+  "result": {
+    "newVMs": {
+      "tGas3T58KzdjLHhBDMnH2TvrddhqTji5iZAMZ3RXs2NLpSnhH": [
+        "timestampvm",
+        "timestamp"
+      ],
+      "spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ": []
+    }
+  },
+  "id": 1
+}
+```
+
+Now, this VM's static API can be accessed at endpoints `/ext/vm/timestampvm` and
+`/ext/vm/timestamp`.  For more details about VM configs, see
+[here](../nodes/maintain/avalanchego-config-flags.md#vm-configs).
+
```

Comment:
Again for the name of this Timestamp, it should be named differently to timestamp-vs, otherwise, we will have developers trying both this and https://docs.avax.network/subnets/create-a-vm-timestampvm#installing-a-vm and it will create much unneeded confusion and support calls.